current/local-cli/rnpm/windows/index.js

```diff
@@ -15,6 +15,10 @@ module.exports = [{
   }, {
     command: '--template [template]',
     description: 'Template to install. E.g., `vnext`.',
+  }, {
+    command: '--language [language]',
```

Comment:
You are make change to current. do we need to copy rnpm-plugin-windows to vnext and publish as next package too?

Reply:
This code is shared between current and vnext.  What is needed is to simply publish a new rnpm-plugin-windows.  @marlenecota is this done manually or automatically?

Reply:
This is published automatically.